projects/storefrontlib/src/cms-components/cart/cart.module.ts

```diff
@@ -19,6 +19,7 @@ import { MiniCartModule } from './mini-cart/mini-cart.module';
     MiniCartModule,
     CartModule,
   ],
+  declarations: [],
```

Comment:
Redundant

Reply:
Not our file and already remove from what I see in the latest code